yaks/lib/yaks/default_policy.rb

```diff
@@ -5,7 +6,8 @@ class DefaultPolicy
     # Default policy options.
     DEFAULTS = {
       rel_template: "rel:{rel}",
-      namespace: Object
+      namespace: Object,
+      mapper_for: {}
```

Comment:
`mapper_for` is a good name for the DSL method, but I think it's not an ideal name for the variable, IMO `:mappers` would be better.